test/scripts/database.ts

```diff
@@ -3,8 +3,8 @@ const should = chai.should(); // eslint-disable-line
 import path from 'path';
 import Promise from 'bluebird';
 import sinon from 'sinon';
-import Database from '../../dist/database';
-import Model from '../../dist/model';
+import Database from '../../src/database';
+import Model from '../../src/model';
```

Comment:
scripts `preeslint`, `pretest` could be deleted.

https://github.com/hexojs/warehouse/blob/ca7f85184b1931c2128af46b68b908320b408171/package.json#L16
https://github.com/hexojs/warehouse/blob/ca7f85184b1931c2128af46b68b908320b408171/package.json#L18